src/util/symbol.h

```diff
@@ -123,6 +123,12 @@ class symbolt
     return is_auxiliary;
   }
 
+  /// Returns true iff the symbol belongs to the global scope.
+  bool is_global() const
+  {
+    return is_static_lifetime && is_lvalue;
```

Comment:
Well ... except this isn't actually right. This only works for non-type symbols. Maybe call it `is_global_object` and also check `is_type`?

Reply:
And also it doesn't work for code-typed symbols (which aren't lvalues).